api/client/samples/crop_models/sao_paulo_sugar_content/Sao_Paulo_brazil_sugar_content_modeling.ipynb

```diff
@@ -0,0 +1,3628 @@
+{
```

Comment:
<p>this chart includes semi=monthly and annual series points... so the dec 31 value includes the annual value as well.</p><p>before filtering for  the period between 100 and 350, we should simply exclude the annual frequency from the beginning. ANd then see  if further filtering is needed of the points </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gro-intelligence/api-client/pull/76/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>Fixed</p>